src/components/TodosFiltered/TodosFiltered.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  setTodos: (a: Todo[]) => void;
+  allTodos: Todo[];
+};
+
+export const TodosFilter: React.FC<Props> = ({ todos, setTodos, allTodos }) => {
+  enum Status {
+    ALL = 'all',
+    ACTIVE = 'active',
+    COMPLETED = 'Completed',
+  }
+
+  const [filterBy, setFilterBy] = useState<Status>(Status.ALL);
+
+  const getFilteredTodos = async (filter: Status) => {
```

Comment:
The function 'getFilteredTodos' does not need to be async as there are no asynchronous operations in the function.